scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1140,6 +1140,7 @@
         (is (thrown? Exception (s/validate api/Job (assoc min-job :expected-runtime 3 :max-runtime 2))))))))
 
 (deftest test-create-jobs!
+   (util/flush-caches)
```

Comment:
Should a similar call not exist for all other unit tests that trigger these cache operations? Alternatively, if each test creates its own job, should there be no cache hits so there should be no need to flush the caches?

Reply:
Most unit tests get it done via invoking restore-fresh-database! in testutils.clj, which now flushes the caches.